### PR TITLE
More occ user management commands

### PR DESCRIPTION
**Pull request by icewind1991**
Adds the following occ commands
- `occ user:info <user>`: Show user details
- `occ user:list [--limit <limit>] [--offset <offset>`: List users
- `occ group:list [--limit <limit>] [--offset <offset>`: List groups
- `occ group:adduser <group> <user>`: Add a user to a group
- `occ group:removeuser <group> <user>`: Remove a user from a group

### CODE REVIEW

File: core/Command/Group/AddUser.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @author Robin Appelman <robin@icewind.nl>
+ *
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
```
Comment (LukasReschke) [inline core/Command/Group/AddUser.php]: ?

File: core/Command/Group/ListCommand.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * @author Robin Appelman <robin@icewind.nl>
+ *
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
```
Comment (LukasReschke) [inline core/Command/Group/ListCommand.php]: ?

File: core/Command/Group/RemoveUser.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @author Robin Appelman <robin@icewind.nl>
+ *
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
```
Comment (LukasReschke) [inline core/Command/Group/RemoveUser.php]: ?

File: core/Command/User/Info.php [hunk 4]
```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @author Robin Appelman <robin@icewind.nl>
+ *
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
```
Comment (LukasReschke) [inline core/Command/User/Info.php]: ?

File: core/Command/User/ListCommand.php [hunk 5]
```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * @author Robin Appelman <robin@icewind.nl>
+ *
+ * @copyright Copyright (c) 2016, ownCloud, Inc.
```
Comment (LukasReschke) [inline core/Command/User/ListCommand.php]: ?

**Comment by icewind1991** [comment]
@LukasReschke fixed the headers

File: core/Command/Group/AddUser.php [hunk 6]
```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @author Robin Appelman <robin@icewind.nl>
+ *
+ * @license AGPL-3.0
```
Comment (nickvergessen) [inline core/Command/Group/AddUser.php]: Please use the same header everyone else is using:
https://github.com/nextcloud/server/blob/master/apps/theming/tests/lib/TemplateTest.php#L2-L20

Comment (icewind1991) [inline core/Command/Group/AddUser.php]: done

**Comment by MorrisJobke** [comment]
Tested and works 👍 


(I rebased this to resolve the conflict)


@blizzz @schiessle @nickvergessen @LukasReschke Please review

**Comment by MorrisJobke** [comment]
> @blizzz @schiessle @nickvergessen @LukasReschke Please review

and @rullzer mind to review?

**Comment by rullzer** [comment]
:+1:

**Pull request merged**